apax/train/callbacks.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+import tensorflow as tf
+from keras.callbacks import CSVLogger, TensorBoard
+
+log = logging.getLogger(__name__)
+
+
+def initialize_callbacks(callback_configs, model_version_path):
+    log.info("Initializing Callbacks")
+
+    dummy_model = tf.keras.Model()
+    callback_dict = {
+        "csv": {
+            "class": CSVLogger,
+            "log_path": model_version_path / "log.csv",
+            "path_arg_name": "filename",
+            "kwargs": {"append": True},
+            "model": dummy_model,
+        },
+        "tensorboard": {
+            "class": TensorBoard,
+            "log_path": model_version_path,
+            "path_arg_name": "log_dir",
+            "kwargs": {},
+            "model": dummy_model,
+        },
+    }
+
+    callback_configs = [config.name for config in callback_configs]
+    if "csv" in callback_configs and "tensorboard" in callback_configs:
+        csv_idx, tb_idx = callback_configs.index("csv"), callback_configs.index(
+            "tensorboard"
+        )
+        msg = (
+            "Using both csv and tensorboard callbacks is not supported at the moment."
+            " Only the first of the two will be used."
+        )
+        print("Warning: " + msg)
+        log.warning(msg)
```

Comment:
Why are you using both?

Reply:
I am not terribly happy with our logging setup. Log level is set to warn by default and logs are written to a file. I would like to do something like write log.info to a file and log.warning to both stdout and the log file.
Reason being that I think users encountering a warning might not check the log file to see what's going on, but if it's in the terminal, they are going to see it.

Reply:
but I'm not sure what the best way to handle this is